cpp/tests/binaryop/binop-compiled-test.cpp

```diff
@@ -580,6 +580,40 @@ auto NullOp_Result(column_view lhs, column_view rhs)
   return column_wrapper<TypeOut>(result.cbegin(), result.cend(), result_mask.cbegin());
 }
 
+// Yes this is ugly but these logical tests need some things from nullops to work, and this
+// kept the changes smallest
```

Comment:
Instead of saying "ugly" can you explain what is "ugly" about them?

Reply:
I moved the test and its dependencies to be grouped with the other logical tests. I thought that was cleaner instead of explaining why I wanted to keep my patch small in the code itself.